src/app_service/service/community/service.nim

```diff
@@ -629,7 +629,7 @@ QtObject:
             self.chatService.updateChannelMembers(chat)
 
       # members list was changed
-      if (community.isMember or community.tokenPermissions.len == 0) and community.members != prev_community.members:
+      if community.members != prev_community.members:
```

Comment:
We must not show community members in closed communities

Reply:
Members of closed communities will be empty for those who are not members. We have the community description encrypted in that case.

Reply:
This deleted condition was just a UI "hack".